source/catalog.html

```diff
@@ -4,235 +4,373 @@
 <head>
   <meta charset="utf-8">
   <title>HTML Academy: Кэт энерджи</title>
+  <link rel="stylesheet" href="/source/css/style.css">
```

Comment:
Файл стилей не подключился. Нужно исправить путь к нему:
```suggestion
  <link rel="stylesheet" href="css/style.css">
```